api/api/tests/notes_tests.py

```diff
@@ -75,87 +75,6 @@ def test_submit_good_nmap(self) -> None:
-            Device type: general purpose
-            Running: Linux 2.6.X
-            OS details: Linux 2.6.13 - 2.6.31, Linux 2.6.18
-            Network Distance: 13 hops
-
-            TRACEROUTE (using port 80/tcp)
-            HOP RTT       ADDRESS
-            [Cut first 10 hops for brevity]
-            11  45.16 ms  layer42.car2.sanjose2.level3.net (4.59.4.78)
-            12  43.97 ms  xe6-2.core1.svk.layer42.net (69.36.239.221)
-            13  45.15 ms  scanme.nmap.org (64.13.134.52)
-
-            OS and Service detection performed. Please report any incorrect results at https://nmap.org/submit/ .
-            # Nmap done at Sun Jul 18 15:33:48 2010 -- 1 IP address (1 host up) scanned in 22.47 seconds
-        """
-
-        response = self.pentester_client.post('/nmap', format='json', data={
-            'nmap_file': nmap_file_content,
-            'recon_id': self.mission.recon.id,
-            })
-        self.assertEqual(response.status_code, 201)
-
-        id = response.json()['id']
-        nmap_file_content = """
-            Nmap 5.35DC18 scan initiated Sun Jul 18 15:33:26 2010 as: ./nmap -T4 -A -oN - scanme.nmap.org
-            Nmap scan report for scanme.nmap.org (64.13.134.52)
-            Host is up (0.045s latency).
-            Not shown: 993 filtered ports
-            PORT      STATE  SERVICE VERSION
-            23/tcp    open   ssh     OpenSSH 4.3 (protocol 2.0)
-            | ssh-hostkey: 1024 60:ac:4d:51:b1:cd:85:09:12:16:92:76:1d:5d:27:6e (DSA)
-            |_2048 2c:22:75:60:4b:c3:3b:18:a2:97:2c:96:7e:28:dc:dd (RSA)
-            1337/tcp    closed smtp
-            53/tcp    open   domain
-            70/tcp    closed gopher
-            80/tcp    open   http    Apache httpd 2.2.3 ((CentOS))
-            | http-methods: Potentially risky methods: TRACE
-            |_See https://nmap.org/nsedoc/scripts/http-methods.html
-            |_html-title: Go ahead and ScanMe!
-            113/tcp   closed auth
-            31337/tcp closed Elite
-            Device type: general purpose
-            Running: Linux 2.6.X
-            OS details: Linux 2.6.13 - 2.6.31, Linux 2.6.18
-            Network Distance: 13 hops
-
-            TRACEROUTE (using port 80/tcp)
-            HOP RTT       ADDRESS
-            [Cut first 10 hops for brevity]
-            11  45.16 ms  layer42.car2.sanjose2.level3.net (4.59.4.78)
-            12  43.97 ms  xe6-2.core1.svk.layer42.net (69.36.239.221)
-            13  45.15 ms  scanme.nmap.org (64.13.134.52)
-
-            OS and Service detection performed. Please report any incorrect results at https://nmap.org/submit/ .
-            # Nmap done at Sun Jul 18 15:33:48 2010 -- 1 IP address (1 host up) scanned in 22.47 seconds
-        """
-        response = self.pentester_client.patch(f'/nmap/{id}', format='json', data={
-            'nmap_file': nmap_file_content,
-            'recon_id': self.mission.recon.id,
-            })
-        self.assertEqual(response.status_code, 200)
-
```

Comment:
Can you please update the test rather than deleting it

Reply:
![image](https://github.com/vrn-sh/erp/assets/29751352/76da1a23-dd2c-4e9e-9b69-d28d34493790)

no